site/docs/tools/iceberg/hive.md

```diff
@@ -61,7 +61,7 @@ Whereby the above properties are explained as below:
 
 ## Writing and reading tables
 
-To read and write into tables that are managed by Iceberg and Nessie, typical Hive SQL queries can be used. Refer to this documentation [here](https://iceberg.apache.org/hive/#querying-with-sql) for more information.
+To read and write into tables that are managed by Iceberg and Nessie, typical Hive SQL queries can be used. Refer to this documentation [here](https://iceberg.apache.org/docs/nightly/hive/#create-external-table-overlaying-an-existing-iceberg-table) for more information.
```

Comment:
Couldn't find the similar url https://iceberg.apache.org/hive/#querying-with-sql .

Reply:
That anchor has been deleted in 2022 :-D 

https://github.com/apache/iceberg/commit/dd5b35067baac5eb004141d51d32251d2f8e808c 

The link now should be: https://iceberg.apache.org/docs/latest/hive/#dml-commands.